docs/windows.md

```diff
@@ -2,11 +2,142 @@
+
+#### Install Visual Studio 2019
+With your admin PowerShell terminal:
+
+```PowerShell
+choco install visualstudio2019-workload-vctools
+```
+
+#### Install Python 3.6
+
+1. Download [Python 3.6.8](https://www.python.org/ftp/python/3.6.8/python-3.6.8-amd64-webinstall.exe)
+1. Launch the installer by double clicking on it
+1. On the first screen of the installation wizard, leave the default options and
+select Customize installation
+1. On the Optional Features screen, select Next to keep the defaults
+1. On the Advanced Options screen, unselect the two selected options
+1. In the Customize install location field enter `C:\Python36` and select Install
+1. Allow the installation to finish
+
+#### Install gvsbuild
+We have forked gvsbuild for now because the upstream project is a little slow at
+accepting our Pull Requests. If we can get our changes upstream, then we'll
+switch back to using that version directly.
+
+Open a new regular user PowerShell terminal and execute:
+
+```PowerShell
+mkdir C:\gtk-build\github
+cd C:\gtk-build\github
+git clone https://github.com/gaphor/gvsbuild.git
+
+```
+
+#### Build GTK
+
+In the same PowerShell terminal, execute:
+
+```PowerShell
+C:\Python36\python.exe .\build.py build -p=x64 --vs-ver=16 --msys-dir=C:\tools\msys64 --enable-gi --py-wheel --gtk3-ver=3.24 gobject-introspection gtk3 pycairo pygobject adwaita-icon-theme hicolor-icon-theme
+```
+Grab a coffee, the build will take a few minutes to complete.
+
+Once it is complete, add GTK to your path:
+```PowerShell
+$env:Path = "C:\gtk-build\gtk\x64\release\bin;" + $env:Path
+```
+
+#### Install the Latest Python for Gaphor
+
+Download and install the latest version of Python. For this you have a few options:
+
+1. Install from Chocolately with `choco install python` as an admin 
+1. Install from the Windows Store
+1. If you need to test with multiple versions of Python, install
+[pyenv-win](https://github.com/pyenv-win/pyenv-win)
+
+Restart your PowerShell terminal as a normal user and check that `python --version` is correct.
+
+#### Setup Gaphor
+
+In the same PowerShell terminal, clone the repository:
+```PowerShell
+cd (to the location you want to put Gaphor)
+git clone https://github.com/gaphor/gaphor.git
+```
+
+Create and activate a virtual environment in which Gaphor can be installed.
+```PowerShell
+PS > cd gaphor
+PS > python -m venv .venv
+PS > .\.venv\Scripts\activate.ps1
+```
+
+Install and configure Poetry
+```PowerShell
+(.venv) PS > pip install poetry
+(.venv) PS > poetry config virtualenvs.create false
+```
+
+Install PyGObject and pycairo from gvsbuild
+```PowerShell
+(.venv) PS > pip install C:\gtk-build\build\x64\release\pygobject\dist\PyGObject-*-cp39-cp39-win_amd64.whl
+(.venv) PS > pip install C:\gtk-build\build\x64\release\pycairo\dist\pycairo-*-cp39-cp39-win_amd64.whl
+```
+
+Install Gaphor's other dependencies and give it a try
+```PowerShell
+(.venv) PS > poetry install
+(.venv) PS > gaphor
```

Comment:
Gaphor errored initially: I started a new console and forgot to se the path again.

Reply:
Do you think we need to do something more permanent to have the user keep the path?

Reply:
We could move the `$env:Path` down to this section, where it's required. A small notice would do as well, I suppose.

Reply:
Sounds good, update the PR!